example.yaml

```diff
@@ -1,8 +1,8 @@
 state_machines:
   user_lifecycle:
     feeds:
-      - path: jacquard
-        feed: http://localhost:1212/<label_id>
+      - name: jacquard
```

Comment:
Could we cherry-pick this to `master`? (I'm assuming that it doesn't relate to changes on this branch)

Reply:
This is merging now anyway.